sdk/samples/iot/CMakeLists.txt

```diff
@@ -137,6 +137,51 @@ target_link_libraries(paho_iot_hub_twin_sample
 
 create_map_file(paho_iot_hub_twin_sample paho_iot_hub_twin_sample.map)
 
+# Twin CBOR Sample - Only compile if intel/tinycbor has been found
+find_library(tinycbor tinycbor
+  if(WIN32)
+    PATHS $ENV{Path}
+    PATH_SUFFIXES tinycbor/lib lib
+  endif()
+)
+
+if (${tinycbor} STREQUAL "tinycbor-NOTFOUND")
+  message("-- Could NOT find tinycbor. Not compiling paho_iot_hub_twin_cbor_sample")
+else()
+  if(MSVC)
+    set(CMAKE_EXE_LINKER_FLAGS
+        "${CMAKE_EXE_LINKER_FLAGS} /NODEFAULTLIB:LIBCMT.lib")
+  endif()
+
+  add_executable (paho_iot_hub_twin_cbor_sample
+    ${CMAKE_CURRENT_LIST_DIR}/paho_iot_hub_twin_cbor_sample.c
+  )
+
+  target_link_libraries(paho_iot_hub_twin_cbor_sample
+    PRIVATE
+      az::iot::sample::common
+  )
+
+  target_link_libraries(paho_iot_hub_twin_cbor_sample
+    PRIVATE
+      ${tinycbor} #Required to be a string for Windows
+  )
+
+  find_path(TINYCBOR_INCLUDE_DIR cbor.h
+    PATHS
+      if(UNIX) /usr/local/include
+      endif()
+
+    PATH_SUFFIXES
+      if(WIN32) tinycbor/src src
+      elseif(UNIX) tinycbor
+      endif()
+  )
+  include_directories(${TINYCBOR_INCLUDE_DIR})
```

Comment:
should this be 
```suggestion
  target_include_directories(paho_iot_hub_twin_cbor_sample PRIVATE ${TINYCBOR_INCLUDE_DIR})
```

Reply:
So i've fixed this and committed it: 976ce6d
But don't know why its not showing up in the PR

Reply:
https://github.com/momuno/azure-sdk-for-c/commit/976ce6d3264009adb7094e08db61d9b9cf156560